lib/matplotlib/font_manager.py

```diff
@@ -1117,12 +1118,11 @@ def score_family(self, families, family2):
                 options = [x.lower() for x in options]
                 if family2 in options:
                     idx = options.index(family2)
-                    return ((0.1 * (idx / len(options))) *
-                            ((i + 1) / len(families)))
+                    return (i + (idx / len(options))) * step
```

Comment:
Also, part of the current bug this PR attempts to fix is that an alias match will return 0 if the font is first in the alias list, regardless of where that alias appears in the font families list.